sections/04_study.md

```diff
@@ -586,23 +576,23 @@ for dealing with batch effects [@tag:Shaham2016_batch_effects].
 
 Examining populations of single cells can reveal biologically meaningful subsets
 of cells as well as their underlying gene regulatory networks
-[@tag:Gaublomme2015_th17]. Unfortunately, machine learning generally struggles
+[@tag:Gaublomme2015_th17]. Unfortunately, machine learning methods generally struggle
 with imbalanced data — when there are many more examples of class 1 than class 2 —
```

Comment:
Looks like a single hyphen not a long dash. Suggest this could all be cleaned up near end with a simple search and replace.